stripe/src/main/java/com/stripe/android/model/PaymentIntent.java

```diff
@@ -427,18 +402,7 @@ public static NextActionType fromCode(@Nullable String code) {
         RequiresConfirmation("requires_confirmation"),
         RequiresPaymentMethod("requires_payment_method"),
         Succeeded("succeeded"),
-
-        /**
-         * @deprecated use {@link #RequiresPaymentMethod}
-         */
-        @Deprecated
-        RequiresSource("requires_source"),
-
-        /**
-         * @deprecated use {@link #RequiresAction}
-         */
-        @Deprecated
-        RequiresSourceAction("requires_source_action");
+        RequiresSource("requires_source");
```

Comment:
@jeffchan-stripe can I remove this enum value as well?

Reply:
Yes, as long as users are making requests will the API version `2019-02-11` or after, then they will not see `requires_source`.

Reply:
Yes, we are now pinning to the latest API version so we won't need `requires_source`